<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>241</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
 qproperty-alignment: 'AlignVCenter | AlignRight';
 border: 1px solid gray;
background-color : #FDE2E2;
border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: crimson;
 border: 1px solid gray;
   border-radius: 5px;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>ะก</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusMinus">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>61</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: crimson;
 border: 1px solid gray;
   border-radius: 5px;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: crimson;
 border: 1px solid gray;
   border-radius: 5px;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: crimson;
 border: 1px solid gray;
   border-radius: 5px;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subtract">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: crimson;
 border: 1px solid gray;
  border-radius: 5px;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decimal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #b28fe6, stop: 1 #e68fc1);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: crimson;
 border: 1px solid gray;
   border-radius: 5px;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: crimson;
 border: 1px solid gray;
   border-radius: 5px;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="top_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
 qproperty-alignment: 'AlignVCenter | AlignRight';
 border: 1px solid gray;
background-color : #FDE2E2;
border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>โ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>xยฒ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_powe">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_factorial">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>x!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: crimson;
 border: 1px solid gray;
   border-radius: 5px;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>โ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fraction">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>deg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 background-color: black;
 border: 1px solid gray;
   border-radius: 5px;
color: white;
}
QPushButton:pressed {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
stop: 0 #F3184B, stop: 1 #FF9AFF);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
